tests/ui/asan-odr-win/asan_odr_windows.rs

```diff
@@ -0,0 +1,15 @@
+//@ run-pass
+//@ compile-flags:-Zsanitizer=address
+//@ aux-build:asan_odr_win-2.rs
```

Comment:
Suggestion: you can use `//@ aux-crate: othercrate=asan_odr_win-2.rs` to specify the crate name, and then you can drop `#![crate_name = "othercrate"]` in the auxiliary (so it's more obvious).

Reply:
Should be fixed now, thanks for the input!